### PR TITLE
New changelog and version bump for 1.0.2

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.2
+ - prevents multiple queries from being run at the same (#70)
```
Comment (untergeek) [inline CHANGELOG.md]: at the same "time"?

being run at once?

being run simultaneously?

Comment (suyograo) [inline CHANGELOG.md]: hah, will fix :)

**Comment by suyograo** [comment]
@untergeek updated

**Comment by untergeek** [comment]
LGTM™

**Pull request closed without merging**